Lib/test/eintrdata/eintr_tester.py

```diff
@@ -71,18 +71,14 @@ def tearDown(self):
         if hasattr(faulthandler, 'cancel_dump_traceback_later'):
             faulthandler.cancel_dump_traceback_later()
 
-    def subprocess(self, *args, **kw):
-        cmd_args = (sys.executable, '-c') + args
-        return subprocess.Popen(cmd_args, **kw)
-
 
 @unittest.skipUnless(hasattr(signal, "setitimer"), "requires setitimer()")
 class OSEINTRTest(EINTRBaseTest):
     """ EINTR tests for the os module. """
 
     def new_sleep_process(self):
         code = 'import time; time.sleep(%r)' % self.sleep_time
-        return self.subprocess(code)
+        return spawn_python('-c', code)
```

Comment:
I don't see how this is beneficial.  The point of that method is to avoid having to type '-c' everywhere.

Reply:
> This doesn't materially improve the eintrdata test suite and the issue it links to is about script_helper, _assert_python, and text mode to the child. which don't seem to be a problem in eintrdata_tester.py

ok, the spawn_python in script_helper is good enough. Forget this.